<launch>

  <arg name="tf_prefix"/>

  <node pkg="csqmi_exploration" type="exploration_manager" 
    name="exploration_manager" output="screen">
    <remap from="vel_cmds" to="velocity_commands"/>
    <param name="tf_prefix" value="$(arg tf_prefix)"/>
  </node>

</launch>
